src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -108,20 +108,31 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
             const auto guidSubStr = std::wstring_view{ wsGuid }.substr(1);
 
             // Ensure every connection has the unique identifier in the environment.
-            environment.insert_or_assign(L"WT_SESSION", guidSubStr.data());
+            environment.insert_or_assign(L"WT_SESSION", guidSubStr.data());            
```

Comment:
You didn't want to add this trailing whitespace
```suggestion
            environment.insert_or_assign(L"WT_SESSION", guidSubStr.data());
```